ci/prow/testgrid.yaml

```diff
@@ -318,6 +322,9 @@ dashboards:
   - name: nightly
     test_group_name: ci-knative-client-nightly-release
     base_options: "sort-by-name="
+  - name: tekton
```

Comment:
Sorry, I can't locate the job at https://testgrid.knative.dev/client.
How can I look at the job/logs?
@adrcunha @chizhg

Reply:
It's been running: https://prow.knative.dev/?job=ci-knative-client-tekton

Looks like the TestGrid config wasn't updated. I'll do it now.

Reply:
It's now up: https://testgrid.knative.dev/client#tekton

Reply:
Thanks! Looking at fixing the job failure..